model/patch/db.go

```diff
@@ -83,14 +83,25 @@ func ById(id mgobson.ObjectId) db.Q {
 	return db.Query(bson.M{IdKey: id})
 }
 
+var commitQueueFilter = bson.M{"$ne": evergreen.CommitQueueAlias}
+
 // ByProject produces a query that returns projects with the given identifier.
-func ByProject(project string) db.Q {
-	return db.Query(bson.M{ProjectKey: project})
+func ByProject(project string, filterCommitQueue bool) db.Q {
+	q := bson.M{ProjectKey: project}
+	if filterCommitQueue {
```

Comment:
This seems unconnected to the ByProject and ByUser queries.
Maybe we could set the Alias predicate on the return value in the [callers](https://github.com/evergreen-ci/evergreen/blob/3af9096f5f3bc045f777ddfc71e28efe85a86399/service/timeline.go#L118)?

On the other hand, it seems like this is the only place we use these functions for the time being.

Reply:
I can rename the functions to be more clear